theano/sandbox/cuda/__init__.py

```diff
@@ -582,13 +579,13 @@ def use(device,
                     if dnn_available():
                         (hdr_v, runtime_v) = dnn_version()
                         cudnn_version = runtime_v
-                        # 4100 should not print warning with cudnn 4 final.
-                        if cudnn_version > 4100:
+                        # 5100 should not print warning with cudnn 5 final.
+                        if cudnn_version > 5100:
                             warn = ("Your CuDNN version is more recent then Theano."
                                     " If you see problems, try updating Theano or"
-                                    " downgrading CuDNN to version 4.")
```

Comment:
cuDNN is a proper name and should not be written with a capital C.
See: https://developer.nvidia.com/cudnn